typescript-timers/interval.ts

```diff
@@ -0,0 +1,11 @@
+const $countDown = document.querySelector('.countdown-display');
+if (!$countDown) throw new Error('countDown query failed');
+let count = 4;
+const i = setInterval(function () {
+  count--;
+  $countDown.textContent = count.toString();
```

Comment:
Do we always want this to happen? By doing this, you are going to show the number for a slight second before changing it to the actual text. We only want to show the number if count is greater than 0